setup.py

```diff
@@ -112,9 +85,7 @@ def run(self):
     },
 
     install_requires=[
-        # Any changes here must be reflected in ``python-flocker.spec.in`` so
-        # that RPM dependencies match.
-        # They must also be reflected in the yumdownloader lines in
+        # Any changes here must be reflected in the yumdownloader lines in
```

Comment:
All of the packages used with the `yumdownloader` commands in the `Pre-populating RPM Repository` currently appear to be Python packages.  Does that mean we don't currently need any of the packages in that RPM repository and until we introduce a non-Python dependency we won't need to do any pre-populating?